File: samples/pedir-idade-e-nome.cs
int idade;
string ?nome;

Console.Clear();

Console.WriteLine("Qual é o seu nome?");
nome = Console.ReadLine();
if (nome == null) {
  nome = "Tudo bem então, Fulano!";
}

Console.WriteLine("Qual a sua idade?");
idade = Convert.ToInt16(Console.ReadLine());

string message;
if (idade < 18) {
  message = "Ainda muito jovem, seu goiaba.";
}
else if (idade >= 18 && idade < 60) {
  message = "Aí simmm";
}
else {
  message = "Vish...";
}
Console.WriteLine(message);

File: samples/pedir-idade-e-nome.cpp
#include <iostream>
#include <string>

// Verificador de idades, para definir uma mensagem para cada idade.
main()
{
	int age;
	std::string name;
	
	std::cout << "Qual seu nome? \n";
	std::getline(std::cin, name);
	fflush(stdin);
	
	std::cout << "Qual a sua idade? \n";
	std::cin >> age;
	fflush(stdin);
	
	if (age < 18) {
		std::cout << "Ainda muito jovem, meu cumpadi. Vai pra casa!";
	}
	else if (age >= 18 && age < 60) {
		std::cout << "Aí simmmmm!";
	}
	else {
		std::cout << "Vish...";
	}
	
	return 0;
}
